src/tdd/Calculator.java

```diff
@@ -0,0 +1,43 @@
+package tdd;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+
+public class Calculator {
```

Comment:
Look at proposed changes in attachment, tests would have to be adjusted to account for corrected totals
[Calculator.txt](https://github.com/matthewchoat/tdd/files/4954992/Calculator.txt)